Ktisis/Interface/Editor/Properties/ActorPropertyList.cs

```diff
@@ -71,7 +89,111 @@ private void DrawActorTab(ActorEntity actor) {
+							gaze.Other.Gaze.Mode = GazeMode.Disabled;
+						}
+					}
+					IsLinked = !IsLinked;
+				}
+				ImGui.SameLine(0, spacing);
+				ImGui.Text(IsLinked ? "Linked" : "Unlinked");
+				ImGui.Spacing();
+			}
+
+			if (IsLinked || !isHuman)
+				result |= DrawGaze(actor, ref gaze.Other.Gaze, GazeControl.All);
+			else {
+				result |= DrawGaze(actor, ref gaze.Eyes.Gaze, GazeControl.Eyes);
+				ImGui.Spacing();
+				result |= DrawGaze(actor, ref gaze.Head.Gaze, GazeControl.Head);
+				ImGui.Spacing();
+				result |= DrawGaze(actor, ref gaze.Torso.Gaze, GazeControl.Torso);
+			}
+
+			if (result)
+				actor.Gaze = gaze;
+		}
+
+		return;
+	}
+
+	private unsafe bool DrawGaze(ActorEntity actor, ref Gaze gaze, GazeControl type) {
+		if (!GazeTables.ContainsKey(type))
+			GazeTables.Add(type, new TransformTable(this._cfg));
+
+		using var _ = ImRaii.PushId($"Gaze_{type}");
+		var spacing = ImGui.GetStyle().ItemInnerSpacing.X;
+
+		var result = false;
+		var enabled = gaze.Mode != 0;
+		var actorCharacter = (CharacterEx*)actor.Character;
+		var isTracking = gaze.Mode == GazeMode._KtisisFollowCam_;
+
+		if (type != GazeControl.All || !enabled) {
+			// if we're changing individual gazes (or viewing All w/o activation), set each to the basegaze for that type
+			// for all when not enabled, this will zero it out
+			var baseGaze = actorCharacter->Gaze[type];
+			gaze.Pos = baseGaze.Pos;
+		} else {
+			// if we're changing ALL gazes and enabled, load in any as theyre all equal to gaze.Pos for the transformtable
+			var baseGaze = actorCharacter->Gaze[GazeControl.Torso];
+			gaze.Pos = baseGaze.Pos;
+		}
+
+		if (ImGui.Checkbox($"{type}", ref enabled)) {
+			result = true;
+			gaze.Mode = enabled ? GazeMode.Target : GazeMode.Disabled;
+		}
+
+		// calc button space for camera and gizmo buttons
+		var btnSpace = Icons.CalcIconSize(FontAwesomeIcon.Eye).X
+			+ Icons.CalcIconSize(FontAwesomeIcon.LocationArrow).X
+			+ spacing * 3;
+		ImGui.SameLine(0, spacing);
+		ImGui.SameLine(0, ImGui.GetContentRegionAvail().X - btnSpace);
+
+		// camera tracking - when pressed, toggle enabled and change gaze mode to KtisisFollowCam (or revert to Target mode)
+		using (ImRaii.PushColor(ImGuiCol.Button, ImGui.GetColorU32(ImGuiCol.ButtonActive), isTracking)) {
+			if (Buttons.IconButtonTooltip(FontAwesomeIcon.Eye, "Camera Tracking", Vector2.Zero)) {
+				result = true;
+				enabled = true;
+				gaze.Mode = isTracking ? GazeMode.Target : GazeMode._KtisisFollowCam_;
+			}
+		}
+		ImGui.SameLine(0, spacing);
+
+		// TODO: gizmo, needs work to generate a new one w/o using ObjectWindow/OverlayWindow
+		// 	or, possible to create a dummy scene object (GazeTarget?) to hijack overlay gizmo?
+		using (var _disable = ImRaii.Disabled()) {
+			using (ImRaii.PushColor(ImGuiCol.Button, ImGui.GetColorU32(ImGuiCol.ButtonActive), isTracking)) {
+				if (Buttons.IconButtonTooltip(FontAwesomeIcon.LocationArrow, "Gizmo Tracking [TODO]", Vector2.Zero)) {}
+			}
+		}
```

Comment:
```suggestion
		using (ImRaii.Disabled()) {
			using var _ = ImRaii.PushColor(ImGuiCol.Button, ImGui.GetColorU32(ImGuiCol.ButtonActive), isTracking)
			if (Buttons.IconButtonTooltip(FontAwesomeIcon.LocationArrow, "Gizmo Tracking [TODO]", Vector2.Zero)) {}
		}
```

to reduce nesting for now.